1.preprocessing-data/ICutils.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import numpy as np
+import pathlib
+from matplotlib import pyplot as plt
+from pathlib import Path
+import os
+import skimage
+
+# explicit import to PyBaSiC due to not having package support
+sys.path.append("./PyBaSiC/")
+import pybasic
+
+def load_pybasic_data(
+    channel_path: pathlib.Path,
+    channel: str,
+    file_extension: str = ".tif",
+    verbosity: bool = True,
+):
+    """
+    Load all images from a specified directory in preparation for pybasic illum correction
+
+    Parameters:
+    channels_path (pathlib.Path): path to directory where the all images for each channel are stored
+    channel (str): The name of the channel (currently one of ["GFP", "RFP", "DAPI"])
+    file_extension (str): The filename extension of the types of files to search for (default: ".tif")
+    verbosity (bool): Prints out information regarding the function running through the images (default: "True")
+
+    Returns:
+        channel_images: list of ndarrays of the loaded in images
+        image_files: list of strings of the paths to each image
+    """
+    image_files = (
+        []
+    )  # List that holds all of the paths to the images in the directory for a specific channel
+    image_names = (
+        []
+    )  # List that holds the names (str) for the images that is used for saving
```

Comment:
```suggestion
     # List that holds the names (str) for the images that is used for saving
     image_names = ([])
```

See what [PEP8 says about in-line comments](https://peps.python.org/pep-0008/#inline-comments).

> Use inline comments sparingly.
>
 > An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.

I think we should use a dedicated line for these comments (above the variable definition) as I have suggested here and above.

Reply:
Agreed, I just left it since it was what black did to it.

@gwaybio Should I still format the documents with black and then just manually change the inline comments?

Reply:
Please follow PEP8 for comment convention (use in-line comments sparingly).

The procedure could be:

1. Run black
2. See if black made any wonky change like this
3. Fix comment to PEP8 convention (sometimes we can't always catch our PEP8 violations! There are things called "linters" which can be configured to flagging PEP8 violations, but that is beyond scope :) )
4. Re-run black to fix

All of our python should be blacked.

Reply:
I thought so but I wanted to confirm. I don't use inline comments so I will make sure those are corrected for.